tests/functional/workloads/cnv/test_vm_lifecycle_and_io.py

```diff
@@ -42,8 +45,10 @@ def test_vm_lifecycle_and_io(self, cnv_workload, setup_cnv):
             vm_obj = cnv_workload(
                 volume_interface=vl_if, source_url=constants.CNV_FEDORA_SOURCE
             )
-            vm_obj.run_ssh_cmd(
-                command="dd if=/dev/zero of=/dd_file.txt bs=1024 count=102400"
-            )
-            vm_obj.scp_from_vm(local_path="/tmp", vm_src_path="/dd_file.txt")
+
+            run_fio(vm_obj, filename=file_path, fio_service_name="fio_test")
+
+            vm_obj.restart()
+            if check_fio_status(vm_obj, fio_service_name):
+                log.info("FIO resumed after restarting VM")
```

Comment:
IIUC, you are restarting the vm, so IO won't be resume. Please update the log message accordingly.

Reply:
Done